### DISCUSSION TITLE
[Feature] Smart Search, People/faces, map view for shared albums.

### DISCUSSION THREAD

**Discussion by Markovski2338** [immich-app/immich#12200] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be very nice for other immich users who you shared albums with to have the smart search, people/faces and map view of the shared photos. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2995/#3176
